tests/dummy/app/pods/view/renderers/models/checkbox-array.js

```diff
@@ -6,6 +6,19 @@ export default {
         type: 'string'
       },
       type: 'array'
+    },
+    serviceAffecting: {
+      items: {
+        type: 'string'
+      },
+      type: 'array'
```

Comment:
Unless you want `['SERVICE_AFFECTING', 'NOT_SERVICE_AFFECTING', 'SERVICE_AFFECTING']` to be valid as well, you probably also want `uniqueItems: true` here

 https://spacetelescope.github.io/understanding-json-schema/reference/array.html#uniqueness